Resources/Prototypes/GameRules/events.yml

```diff
@@ -545,3 +546,28 @@
     maxOccurrences: 1 # this event has diminishing returns on interesting-ness, so we cap it
     weight: 5
   - type: MobReplacementRule
+
+- type: entity
+  parent: BaseGameRule
+  id: DerelictCyborgSpawn
+  components:
+  - type: StationEvent
+    weight: 8
+    earliestStart: 15
+    reoccurrenceDelay: 20
+    minimumPlayers: 4
+    duration: null
+  - type: SpaceSpawnRule
+    spawnDistance: 0
+  - type: AntagSpawner
+    prototype: PlayerBorgDerelict
+  - type: AntagSelection
+    agentName: derelict-cyborg-round-end-agent-name
+    definitions:
+#      briefing:
+#        text: derelict-cyborg-role-greetin
+#        color: Blue
```

Comment:
What's up with this?
Also not sure if the event should count as an antagonist, but I will ask the admins